Paragraphs/Apply-indentation-for-import-content/.NET/Apply-indentation-for-import-content/Program.cs

```diff
@@ -27,6 +27,10 @@
 FileStream outputStream = new FileStream(Path.GetFullPath(@"Output/Output.docx"), FileMode.Create, FileAccess.Write);
 mainDocument.Save(outputStream, FormatType.Docx);
 
+//Dispose the streams.
```

Comment:
Add proper using statement to close file stream and Word document instance

Reply:
addressed